docs/source/testing/mocking.md

```diff
@@ -143,11 +152,13 @@ server.listen().then(({ url }) => {
 });
 ```
 
+Running the above example, with the `mockEntireSchema` option set to false, `resolved` will now return the string `Resolved`.
```

Comment:
Cut comma after "example"

"`resolved` will now return" -> "the `resolved` field now returns"